pages/categories.js

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+function Categories() {
+  return (
+    <>
+      <h1>Categies Site</h1>
+      <Link href="/content/content1">
```

Comment:
it can be confusing to navigate from `/categories` to `/content/content1`

Reply:
*In progress